src/main/resources/application.conf

```diff
@@ -28,6 +28,22 @@ datastax-java-driver {
     }
   }
   basic.request.timeout = 10 seconds
+  advanced.request-tracker {
+    classes = [RequestLogger]
+    logs {
+      success.enabled = false
+      slow {
+        threshold = 1 second
+        enabled = true
+      }
+      error.enabled = true
+      max-query-length = 500
+      show-values = true
```

Comment:
I think we should set this to false so that we don't get any customer data in the logs

Reply:
That does seem safer default choice, +1.

Reply:
Agreed. Changed to `false` in [this commit](https://github.com/stargate/jsonapi/pull/754/commits/b70a3266bab05de5f9c68f73bd0963e3a3e399f2).